### PR TITLE
Add e2e Test coverage for @BotKube notifier showconfig

### LINKED ISSUE #349: Add test coverage for '@BotKube notifier showconfig` command
**Is your feature request related to a problem? Please describe.**

Test coverage is missing for notifier commands

**Describe the solution you'd like**

`@BotKube notifier showconfig` displays resource configuration that BotKube monitors and notifies about. See https://www.botkube.io/usage/#view-botkube-configuration

Integration tests for validating BotKube command responses are implemented here: https://github.com/infracloudio/botkube/blob/develop/test/e2e/command/botkube.go#L46

Add test case to validate  `@BotKube notifier showconfig` command in the existing table drive test cases

### PR DESCRIPTION
**Pull request by anoopmsivadas**
##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY

Add e2e Test coverage for @BotKube notifier showconfig

Fixes #349

### CODE REVIEW

File: test/e2e/command/botkube.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ func (c *context) testBotkubeCommand(t *testing.T) {
 				"  - statefulsets\n" +
 				"  - storageclasses\n",
 		},
+		"BotKube notifier showconfig": {
+			command:  "notifier showconfig",
+			expected: "<@U023BECGF> notifier showconfig",
+		},
```
Comment (PrasadG193) [inline test/e2e/command/botkube.go:89]: Hey @anoopmsivadas,
We also need to verify response as well at https://github.com/infracloudio/botkube/pull/363/files#diff-bf42cad6d13b6d3c0201735d34a4d44aR106.
Let me know if you have any doubts

Comment (anoopmsivadas) [reply test/e2e/command/botkube.go:89]: Hey I accidentally included same commit in another pull request. I don't know how remove this commit from that PR

Comment (anoopmsivadas) [reply test/e2e/command/botkube.go:89]: I will create a new PR with required changes

Comment (PrasadG193) [reply test/e2e/command/botkube.go:89]: @anoopmsivadas We should validate the `expected` output of the command. E.g for `command list`, we are doing it here https://github.com/infracloudio/botkube/blob/aab011cfd60ed85b69380ea5eadebd8df14b64e9/test/e2e/command/botkube.go#L110. Likewise we need to validate the expected output for `notifier showconfig` comamnd

Comment (anoopmsivadas) [reply test/e2e/command/botkube.go:89]: @PrasadG193 OK will do that. but why do we need that switch case there?, its already in a for loop and the filters are same. we could just use the same method used in kubectl test na?

And I'm still figuring out how git and PR works. I shouldn't have deleted the branch gotta create a new PR now i guess

**Pull request closed without merging**